crates/bevy_reflect/src/impls/std.rs

```diff
@@ -1019,6 +1100,27 @@ impl FromReflect for Cow<'static, str> {
     }
 }
 
+impl Typed for &'static Path {
```

Comment:
Why are these impls separated from the `Reflect` impl?

Reply:
No particular reason, I just copied how the other impls in the file are organized.

Reply:
Oh lol then ~~I~~ someone messed up in placing the `Option<T>` impls 🤦

Reply:
It's fine to leave it for now. I'll follow up with a quick cleanup PR!